lib/active_merchant/billing/gateways/omise.rb

```diff
@@ -194,8 +194,10 @@ def post_data(parameters)
 
       def https_request(method, endpoint, parameters=nil, options={})
         raw_response = response = nil
+        req_headers  = headers(options)
```

Comment:
Instead of doing this here, why not modify the `headers` method to add the `Omise-Version` header for you (if available)? You'd be able to get away without modifying the body of this method at all, in that case